apps/deploy-web/tests/import-sdl.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { expect, test } from "@playwright/test";
+import { readFileSync } from "fs";
+import path from "path";
+
+import { BuildTemplatePage } from "./pages/BuildTemplatePage";
+
+test("import sdl", async ({ page, context }) => {
```

Comment:
just to understand. is this related to the PR purpose? I mean according to it's title?